.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1325,6 +1345,39 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             return ret;
         }
 
+        // Parse entities that are made up by two time points with now
+        private List<DateTimeParseResult> GetPrWithNow(MatchCollection matches, ExtractResult er, DateObject referenceDate)

```

Comment:
Rename to ExtractNowAsDate or ParseNowAsDate.

Reply:
I'd prefer Extract, and Parse later, as needed. Similar to the behaviour in the original method.

Reply:
Ok, i will rename the function
 In the Parse stage of the  original method, DateParser couldn't parse the "now" for now is a datetime value, so return a pr about now maybe more simple